CONFIGURATION.md

```diff
@@ -120,7 +120,11 @@ ENV['OTEL_RUBY_INSTRUMENTATION_MYSQL2_CONFIG_OPTS'] = 'db_statement=include;'
 ENV['OTEL_RUBY_INSTRUMENTATION_NET_HTTP_CONFIG_OPTS'] = 'untraced_hosts=localhost,internal.service;'
 ```
 
-> See the [OpenTelemetry Ruby instrumentation documentation](https://opentelemetry.io/docs/languages/ruby/libraries/) for all available options.
+See the [OpenTelemetry Ruby instrumentation documentation](https://opentelemetry.io/docs/languages/ruby/libraries/) for all available options.
+
+#### Exporting Application Logs
+
+The [Logger instrumentation](https://github.com/open-telemetry/opentelemetry-ruby-contrib/tree/main/instrumentation/logger) is a log appender for the Ruby `logger` that bridges messages into OTel log record format for export. By default this instrumentation is disabled, it can be enabled by setting the environment variable `OTEL_RUBY_INSTRUMENTATION_LOGGER_ENABLED=true`.
```

Comment:
Super nit


```suggestion
The [Logger instrumentation](https://github.com/open-telemetry/opentelemetry-ruby-contrib/tree/main/instrumentation/logger) is a log appender for the Ruby `logger` that bridges messages into OTel log record format for export. By default this instrumentation is disabled; it can be enabled by setting the environment variable `OTEL_RUBY_INSTRUMENTATION_LOGGER_ENABLED=true`.
```